package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "s47_bizarre_cravings",
+  "version": "1.0.0",
+  "description": "Hello",
```

Comment:
Add a more detailed description for the project. This will help others understand what the project is about. [important]